.github/workflows/check-dev-containers.yml

```diff
@@ -0,0 +1,49 @@
+name: Check Dev Containers
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.head_ref || github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  containers:
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: network=host
```

Comment:
The qemu and buildx are actually needed to perform a multi-arch build and provide the dev container both for amd64 and arm64. For the verification only I think we can actually spare us these steps in favor of fast builds.